README.md

```diff
@@ -50,7 +51,8 @@ skipper make generate-from-swagger
 
 ### Pre-configuration
 
-- Run minikube on your system.
+- Run minikube on your system. **NOTE: You must enable the registry addon in
+    minikube with `minikube addons enable registry`.**
```

Comment:
Just curious, what's the need for the registry addon?

Reply:
The `deploy-test` target eventually runs `_update_minikube` that builds images on the minikube host. We need the registry addons for this to work properly.

It's hard to remember what combination of `ASSISTED_ORG=minikube-local-registry` or `minikube docker-env` makes the registry necessary. The only thing I do remember is not being able to run through test scenarios using `make` until I enabled it.

Reply:
@YuviGold maybe it can replace our `eval` use?

Reply:
It does. I was checking for this alternative. But I remember I didn't choose it because it requires manual minikube configuration. 

@djzager 
But out of curiosity, why is that better?
You shouldn't call directly to `minikube-local-registry` and `minikube docker-env` while `deploy-test` takes care of all the process.

Reply:
I leave open the possibility that I was missing something when I first tried to go through the README, but I wasn't able to get the `make deploy-test` to work until I enabled the minikube registry config option. I'm happy to remove it and investigate it separately. Sound reasonable?